projects/elonkit/src/ui/avatar/__stories__/avatar.stories.mdx

```diff
@@ -0,0 +1,216 @@
+import { Meta, Story, ArgsTable } from '@storybook/addon-docs/blocks';
+import { Canvas } from '~storybook/components';
+import { action } from '@storybook/addon-actions';
+
+import { MatIconModule } from '@angular/material/icon';
+
+import { ESAvatarComponent, ESAvatarModule } from '..';
+
+import { AVATAR_STORY_IMAGE_SOURCE } from './avatar-story-image';
+
+import { AVATAR_STORY_NO_IMAGE_ICON_SOURCE } from './avatar-story-no-image-icon';
+
+import {
+  AvatarStoryNoImageTypographyModule,
+  AvatarStoryNoImageTypographyComponent,
+  AVATAR_STORY_NO_IMAGE_TYPOGRAPHY_SOURCE
+} from './avatar-story-no-image-typography';
+
+import {
+  AvatarStoryGroupModule,
+  AvatarStoryGroupComponent,
+  AVATAR_STORY_GROUP_SOURCE
+} from './avatar-story-group';
+
+<Meta title='UI/Avatar' />
+
+# Avatar
+
+This component allows to display default or custom avatar.
+
+## Demos
+
+####
+
+####
+
+### Image avatars
+
+Are created by passing inputs:
+
+####
+
+`src` - path to image;
+`alt` - text for alt attribute;
+
+<Canvas source={AVATAR_STORY_IMAGE_SOURCE}>
+  <Story
+    name='Image'
+    args={{
+      src: '/img/es-logo.png',
+      alt: 'alt text',
+      variant: 'round',
+      size: 40
+    }}
+    argTypes={{
+      variant: {
+        control: { type: 'select', options: ['round', 'square'] }
+      }
+    }}
+  >
+    {((args) => ({
+      template: `
+        <es-avatar
+          [size]="size"
+          [src]="src"
+          [alt]="alt"
+          [variant]="variant"
+        ></es-avatar>
+      `,
+      moduleMetadata: {
+        imports: [ESAvatarModule]
+      },
+      props: {
+        ...args
+      }
+    })).bind({})}
+  </Story>
+</Canvas>
+
+### Icon avatars
+
+Are created by passing mat-icon via ng-content instead of src input
```

Comment:
When `src` input is missing or falsy, content is used instead. Icon or text can be used for a content.